lib/classes/Service.js

```diff
@@ -93,6 +93,7 @@ class Service {
         that.plugins = serverlessYml.plugins;
         that.resources = serverlessYml.resources;
         that.functions = serverlessYml.functions;
+        that.runtime = serverlessYml.runtime;
```

Comment:
Not sure if the runtime should be global we've implemented it as a provider specific property.